### ISSUE TITLE
Clarify that once the attempt is started, the result can't be DNS

### ISSUE THREAD

**Issue by Claster** [thewca/wca-regulations#657]
E.g. a widespread misconception is that if a timer wasn't started, the result is DNS, while DNF is correct.

**Comment by xsrvmy** [comment]
I assume there would need to be a clarification for what happens if a competitor declines an extra on an otherwise invalid result (DNF on original result vs DNS for not doing extra)

**Comment by lgarron** [comment]
> [9f4+](https://www.worldcubeassociation.org/regulations/guidelines.html#9f4+)) EXPLANATION The result of an attempt is DNF only if the competitor began the attempt by indicating that they are ready (see Regulation A3b2) but the attempt was disqualified.
> [9f5+](https://www.worldcubeassociation.org/regulations/guidelines.html#9f5+)) CLARIFICATION The result of an attempt is DNS if the competitor was eligible for the attempt and did not start it (see Regulation A3b2). If the competitor did not qualify for an attempt (e.g. in a combined round), they have no result for the attempt.

;-)

**Comment by xsrvmy** [comment]
Should we be more explicit that a DNS should be awarded if a competitor is disqualified from an attempt before it starts? Most common way is probably knocking over the cube cover.
The wording "did not start it" sounds to me like the competitor intentionally forfeited the attempt.

**Comment by Goosly** [comment]
> if a competitor is disqualified from an attempt

The words "disqualification of the attempt" appear in the regulations exactly 32 times. All 32 times, they are followed by "(DNF)". What makes you think a disqualified attempt is recorded as DNS?

Knocking over the cube cover (intentionally or not) is basically cheating.

**Comment by xsrvmy** [comment]
@Goosly firstly I would disagree that knocking over the cube cover is cheating.
Secondly knocking over the cube cover occurs before the start of the attempt. 9f4+ state that the result of an attempt is DNF ONLY IF the competitor began the attempt..., so a DNF penalty cannot be applied. All instances of the wording "disqualification of the attempt" address an attempt that has been started, but this is a case where the attempt has not been started.

**Comment by Goosly** [comment]
From the report of SSL 1 Åkers Styckebruk 2019:

> The cover was accidentally knocked over by the competitor, and since it was the competitors fault, I didn´t award an extra but a DNF.

From the report of Ugine Jeu et Jouet 2018:

> A competitor briefly knocked over the cube cover while setting up their camera. They immediately called me and showed me the footage in which we could clearly see the cube got fully uncovered for a fraction of second. Since it was the competitor's fault and they now clearly saw the cube in the video, DNF was given.

**Comment by zeecho** [comment]
[9f4](https://www.worldcubeassociation.org/regulations/#9f4)) The result of an attempt is recorded as DNF (Did Not Finish) if the attempt is disqualified or unsolved/unfinished.
[9f5](https://www.worldcubeassociation.org/regulations/#9f5)) The result of an attempt is recorded as DNS (Did Not Start) if the competitor is eligible for an attempt but declines it.

When you knock over the cube cover, you don't decline the attempt but you are being disqualified. The term "cheating" used by @Goosly might seem extreme but it's actually close to what it is: you did an action (which is entirely your fault) that made you saw the scramble before the inspection phase, hence your attempt is disqualified (= DNF).

By the way, you'll also get DNFs for the remaining attempts if the Delegate(s) disqualify you from an event, even though you did not start your attempts:
[2j](https://www.worldcubeassociation.org/regulations/#2j)) The WCA Delegate may disqualify a competitor from a specific event.
[2j1](https://www.worldcubeassociation.org/regulations/#2j1)) If a competitor is disqualified from an event for any reason, they are not eligible for any more attempts in the event.
[2j1a](https://www.worldcubeassociation.org/regulations/#2j1a)) The results of all remaining attempts in the event are recorded as DNF.

Basically, you'll get a DNS when you decline an attempt you were allowed to do and that's it. Everything else is DNF.

**Comment by xsrvmy** [comment]
In that case isn't 9f4+ being contradicted directly by 2j1a? I get that the guidelines are supposed to complement the regs but this is a direct contradiction.

Also I said I don't think knocking over the cube cover should not be considered cheating because the delegate zhould not be able to disqualify more than just the individual solve with 2k2.
